tests/test_suite.py

```diff
@@ -109,6 +108,12 @@ def parse_args():
                                    "ask for subset of all rules this way. (If you "
                                    "type ipv6 as a target, all rules containing "
                                    "ipv6 within id will be performed."))
+    parser_rule.add_argument("--manualdebug",
```

Comment:
is there any other debug than "manual" debug? If not, why have the extra adjective there?

Reply:
You are right, it's not necessary to be public facing. I have left manual_debug variable in the code though. I feel it's more clear than having only `debug`.